lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -500,26 +501,6 @@ def setUp(self):
         role.add_users(coach)
         ccx = CcxFactory(course_id=course.id, coach=self.coach)
 
-        # Apparently the test harness doesn't use LmsFieldStorage, and I'm not
-        # sure if there's a way to poke the test harness to do so.  So, we'll
-        # just inject the override field storage in this brute force manner.
-        OverrideFieldData.provider_classes = None
-        # pylint: disable=protected-access
-        for block in iter_blocks(course):
-            block._field_data = OverrideFieldData.wrap(coach, course, block._field_data)
-            new_cache = {'tabs': [], 'discussion_topics': []}
-            if 'grading_policy' in block._field_data_cache:
-                new_cache['grading_policy'] = block._field_data_cache['grading_policy']
-            block._field_data_cache = new_cache
-
-        def cleanup_provider_classes():
-            """
-            After everything is done, clean up by un-doing the change to the
-            OverrideFieldData object that is done during the wrap method.
-            """
-            OverrideFieldData.provider_classes = None
-        self.addCleanup(cleanup_provider_classes)
-
```

Comment:
`iter_blocks` was only actually ever iterating over the course. And the grading code was, in fact, using LmsFieldData (and the OverrideFieldData), so this level of manipulation wasn't necessary.